code/io/keycontrol.cpp

```diff
@@ -692,15 +692,6 @@ void process_debug_keys(int k)
 		case KEY_DEBUGGED + KEY_H:
 			hud_target_toggle_hidden_from_sensors();
 			break;
-
-		case KEY_DEBUGGED + KEY_F: 
-			extern int wacky_scheme;
-			if(wacky_scheme == 3){
-				wacky_scheme = 0;
-			} else {
-				wacky_scheme++;
-			}
-			break;
```

Comment:
Whoops, did git get confused?

Reply:
Yeah, whatever it used to do has been lost to time. I stumbled across it when I removed the console command to alter the break_* values, and when it became clear that it did nothing except get cycled by a cheat.....